File: Front/src/app/Models/Usuarios.ts
import { Contactos } from './Contactos';

export class Usuarios{
    constructor(
        public Id:number,
        public Name:string,
        public LastName:string,
        public UserName:string,
        public Password:string,
        public TypeUser:number,
        public Contactos:Contactos[]
    ){

    }
}

File: Back/AgendaWeb/Models/Usuarios.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AgendaWeb.Models
{
    public class Usuarios
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public int TypeUser { get; set; }
        public List<Contactos> Contacts { get; set; }

    }
}
    